tests/docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+services:
+  irods-server:
+    container_name: irods-server
+    image: wsinpg/ub-18.04-irods-4.2.11
+    restart: always
+    ports:
+      - "1247:1247"
+
```

Comment:
The port range should also include iRODS' higher ports for parallel transfer (2000-20199)